### ISSUE TITLE
NamedArguments rule: false positive with varargs

### ISSUE THREAD

**Issue by t-kameyama** [detekt/detekt#3291]
```kotlin
fun foo(vararg i: Int) {}

fun test() {
    foo(1, 2, 3, 4, 5)
}
```
